dev/config.yml

```diff
@@ -1,12 +1,23 @@
-# Override support for default config entries in defaults.yml.
```

Comment:
Yeah, I set up the basic version then separately added it to ignore to lock in place. Alternatively I could use config.yml.example.. .